### PR TITLE
Autocomplete fixes & imporovements.

### PR DESCRIPTION
**Pull request by ghost**
- fixed tooltip error by used other tooltip than foundation
- reduced requests by concatenating elements (by vulcanize command)
- decreasing build size by removing useless static files

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -58,10 +60,11 @@ gulp.task('js', function(){
         './bower_components/jquery.cookie/jquery.cookie.js',
         './bower_components/jquery/dist/jquery.js',
         './bower_components/modernizr/modernizr.js',
-        './bower_components/foundation/js/foundation.min.js',
+        './bower_components/foundation/js/foundation.js',
```
Comment (mkurek) [inline gulpfile.js]: revert it

File: docker/init_js.sh [hunk 2]
```diff
@@ -3,3 +3,4 @@ set -e
 
 cd $RALPH_DIR
 $RALPH_DIR/node_modules/.bin/gulp
+$RALPH_DIR/node_modules/.bin/gulp polymer-prod
```
Comment (mkurek) [inline docker/init_js.sh]: merge it into one command (run `dev` from `prod`)

File: src/ralph/admin/templates/admin/includes/javascripts.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
 <script type="text/javascript" src="{% static 'vendor/js/jquery.js' %}"></script>
 <script type="text/javascript" src="{% static 'src/js/ralph.init.js' %}"></script>
 <script type="text/javascript" src="{% static "auto-complete-helpers.js" %}"></script>
-<script type="text/javascript" src="{% static 'vendor/js/foundation.min.js' %}"></script>
+<script type="text/javascript" src="{% static 'vendor/js/foundation.js' %}"></script>
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/includes/javascripts.html]: ditto

File: src/ralph/data_importer/resources.py [hunk 4]
```diff
@@ -600,7 +600,7 @@ class OperationResource(RalphModelResource):
         column_name='base_objects',
         attribute='base_objects',
         widget=widgets.ManyToManyWidget(base.BaseObject),
-        default=[],
+        #default=[],
```
Comment (mkurek) [inline src/ralph/data_importer/resources.py]: revert it

File: src/ralph/js_tests/test_runner.html [hunk 5]
```diff
@@ -35,7 +35,7 @@
 
   <script src="../static/vendor/js/jquery.js"></script>
   <script src="../static/src/js/ralph.init.js"></script>
-  <script src="../static/vendor/js/foundation.min.js"></script>
+  <script src="../static/vendor/js/foundation.js"></script>
```
Comment (mkurek) [inline src/ralph/js_tests/test_runner.html]: ditto

File: docker/init_js.sh [hunk 6]
```diff
@@ -2,4 +2,4 @@
 set -e
 
 cd $RALPH_DIR
-$RALPH_DIR/node_modules/.bin/gulp
+$RALPH_DIR/node_modules/.bin/gulp build
```
Comment (mkurek) [inline docker/init_js.sh:5]: @xliiv did you find out why it doesn't run properly in docker?

Comment (ghost) [inline docker/init_js.sh:5]: @mkurek now it works Oo

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**